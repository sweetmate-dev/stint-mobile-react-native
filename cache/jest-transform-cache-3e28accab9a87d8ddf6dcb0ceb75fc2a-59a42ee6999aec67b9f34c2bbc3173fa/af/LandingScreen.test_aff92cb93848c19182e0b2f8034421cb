f07ffbb072914b86b8c2fa4f59fad82a
var _jsxFileName = '/Volumes/Work/ReactNative/stint-mobile-app-dev/src/__tests__/Screens/LandingScreen.test.js';
require('react-native');
var _react = require('react');var _react2 = babelHelpers.interopRequireDefault(_react);
var _LandingScreen = require('../../screens/LandingScreen/LandingScreen');var _LandingScreen2 = babelHelpers.interopRequireDefault(_LandingScreen);

var _reactTestRenderer = require('react-test-renderer');var _reactTestRenderer2 = babelHelpers.interopRequireDefault(_reactTestRenderer);

it('renders correctly', function () {
  var tree = _reactTestRenderer2.default.create(
  _react2.default.createElement(_LandingScreen2.default, { __source: { fileName: _jsxFileName, lineNumber: 10 } })).
  toJSON();
  expect(tree).toMatchSnapshot();
});