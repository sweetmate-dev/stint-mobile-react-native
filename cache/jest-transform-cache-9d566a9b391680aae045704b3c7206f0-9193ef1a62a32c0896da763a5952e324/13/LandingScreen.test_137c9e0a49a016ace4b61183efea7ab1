f28aac62d0c77bad0598ea2e92534e74

require('react-native');
var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _LandingScreen = require('../../screens/LandingScreen/LandingScreen');var _LandingScreen2 = _interopRequireDefault(_LandingScreen);

var _reactTestRenderer = require('react-test-renderer');var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

it('renders correctly', function () {
  var tree = _reactTestRenderer2.default.create(
  _react2.default.createElement(_LandingScreen2.default, null)).
  toJSON();
  expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkxhbmRpbmdTY3JlZW4udGVzdC5qcyJdLCJuYW1lcyI6WyJpdCIsInRyZWUiLCJyZW5kZXJlciIsImNyZWF0ZSIsInRvSlNPTiIsImV4cGVjdCIsInRvTWF0Y2hTbmFwc2hvdCJdLCJtYXBwaW5ncyI6IjtBQUNBO0FBQ0EsOEI7QUFDQSwwRTs7QUFFQSx3RDs7QUFFQUEsR0FBRyxtQkFBSCxFQUF3QixZQUFNO0FBQzVCLE1BQU1DLE9BQU9DLDRCQUFTQyxNQUFUO0FBQ1gsZ0NBQUMsdUJBQUQsT0FEVztBQUVUQyxRQUZTLEVBQWI7QUFHQUMsU0FBT0osSUFBUCxFQUFhSyxlQUFiO0FBQ0QsQ0FMRCIsImZpbGUiOiJMYW5kaW5nU2NyZWVuLnRlc3QuanMiLCJzb3VyY2VzQ29udGVudCI6WyJcbmltcG9ydCAncmVhY3QtbmF0aXZlJztcbmltcG9ydCBSZWFjdCBmcm9tICdyZWFjdCc7XG5pbXBvcnQgTGFuZGluZ1NjcmVlbiBmcm9tICcuLi8uLi9zY3JlZW5zL0xhbmRpbmdTY3JlZW4vTGFuZGluZ1NjcmVlbidcblxuaW1wb3J0IHJlbmRlcmVyIGZyb20gJ3JlYWN0LXRlc3QtcmVuZGVyZXInO1xuXG5pdCgncmVuZGVycyBjb3JyZWN0bHknLCAoKSA9PiB7XG4gIGNvbnN0IHRyZWUgPSByZW5kZXJlci5jcmVhdGUoXG4gICAgPExhbmRpbmdTY3JlZW4gLz5cbiAgICApLnRvSlNPTigpO1xuICBleHBlY3QodHJlZSkudG9NYXRjaFNuYXBzaG90KCk7XG59KTsiXX0=