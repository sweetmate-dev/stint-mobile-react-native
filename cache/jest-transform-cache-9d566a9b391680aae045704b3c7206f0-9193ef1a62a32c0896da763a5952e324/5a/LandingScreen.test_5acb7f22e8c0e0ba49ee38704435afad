36e4e6a2ccade7101522371373297198
"use strict";
require("react-native");
var _react = _interopRequireDefault(require("react"));
var _LandingScreen = _interopRequireDefault(require("../../screens/LandingScreen/LandingScreen"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));var _jsxFileName = "/Users/johnamuesi/Documents/PRODUCTION/REACTNATIVE/STINT/stint/src/__tests__/Screens/LandingScreen.test.js";function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

it('renders correctly', function () {
  var tree = _reactTestRenderer.default.create(
  _react.default.createElement(_LandingScreen.default, { __source: { fileName: _jsxFileName, lineNumber: 10 } })).
  toJSON();
  expect(tree).toMatchSnapshot();
});