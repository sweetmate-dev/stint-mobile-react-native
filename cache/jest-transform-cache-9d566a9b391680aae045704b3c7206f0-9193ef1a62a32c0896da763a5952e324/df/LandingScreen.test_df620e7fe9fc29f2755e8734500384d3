b349e14420112a12da5e2ce2044cf4c1

require('react-native');
var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _LandingScreen = require('../../screens/LandingScreen/LandingScreen');var _LandingScreen2 = _interopRequireDefault(_LandingScreen);

var _reactTestRenderer = require('react-test-renderer');var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

it('renders correctly', function () {
  var tree = _reactTestRenderer2.default.create(
  _react2.default.createElement(_LandingScreen2.default, null));

});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkxhbmRpbmdTY3JlZW4udGVzdC5qcyJdLCJuYW1lcyI6WyJpdCIsInRyZWUiLCJyZW5kZXJlciIsImNyZWF0ZSJdLCJtYXBwaW5ncyI6IjtBQUNBO0FBQ0EsOEI7QUFDQSwwRTs7QUFFQSx3RDs7QUFFQUEsR0FBRyxtQkFBSCxFQUF3QixZQUFNO0FBQzVCLE1BQU1DLE9BQU9DLDRCQUFTQyxNQUFUO0FBQ1gsZ0NBQUMsdUJBQUQsT0FEVyxDQUFiOztBQUdELENBSkQiLCJmaWxlIjoiTGFuZGluZ1NjcmVlbi50ZXN0LmpzIiwic291cmNlc0NvbnRlbnQiOlsiXG5pbXBvcnQgJ3JlYWN0LW5hdGl2ZSc7XG5pbXBvcnQgUmVhY3QgZnJvbSAncmVhY3QnO1xuaW1wb3J0IExhbmRpbmdTY3JlZW4gZnJvbSAnLi4vLi4vc2NyZWVucy9MYW5kaW5nU2NyZWVuL0xhbmRpbmdTY3JlZW4nXG5cbmltcG9ydCByZW5kZXJlciBmcm9tICdyZWFjdC10ZXN0LXJlbmRlcmVyJztcblxuaXQoJ3JlbmRlcnMgY29ycmVjdGx5JywgKCkgPT4ge1xuICBjb25zdCB0cmVlID0gcmVuZGVyZXIuY3JlYXRlKFxuICAgIDxMYW5kaW5nU2NyZWVuIC8+XG4gICk7XG59KTsiXX0=