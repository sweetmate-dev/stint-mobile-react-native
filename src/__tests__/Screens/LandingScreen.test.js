
import 'react-native';
import React from 'react';
import LandingScreen from '../../screens/LandingScreen/LandingScreen'

import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <LandingScreen />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});