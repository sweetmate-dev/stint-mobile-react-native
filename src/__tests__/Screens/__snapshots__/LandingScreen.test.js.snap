// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <View
    onLayout={[Function]}
  >
    <View
      style={
        Object {
          "backgroundColor": "#F8F8F8",
          "borderBottomColor": "#a7a6ab",
          "borderBottomWidth": 0.5,
          "elevation": 3,
          "flexDirection": "row",
          "height": 64,
          "justifyContent": "center",
          "left": 0,
          "paddingLeft": 6,
          "paddingRight": 10,
          "paddingTop": 18,
          "right": 0,
          "shadowColor": undefined,
          "shadowOffset": undefined,
          "shadowOpacity": undefined,
          "shadowRadius": undefined,
          "top": 0,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "flex-start",
            "alignSelf": "center",
            "flex": 1,
          }
        }
      >
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "alignSelf": null,
              "backgroundColor": "transparent",
              "borderBottomWidth": null,
              "borderColor": null,
              "borderLeftWidth": null,
              "borderRadius": 5,
              "borderRightWidth": null,
              "borderTopWidth": null,
              "borderWidth": undefined,
              "elevation": 0,
              "flexDirection": "row",
              "height": 45,
              "justifyContent": "space-between",
              "marginLeft": -3,
              "opacity": 1,
              "paddingBottom": 6,
              "paddingLeft": 4,
              "paddingRight": 6,
              "paddingTop": 6,
              "shadowColor": null,
              "shadowOffset": null,
              "shadowOpacity": null,
              "shadowRadius": null,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 12,
                },
                Array [
                  Object {
                    "color": "#000",
                    "fontSize": 30,
                  },
                  Object {
                    "color": "#007aff",
                    "fontSize": 34,
                    "marginLeft": 1,
                    "marginRight": 2,
                    "marginTop": 0,
                    "paddingTop": 1,
                  },
                ],
                Object {
                  "fontFamily": "Ionicons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            ï‡‚
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "center",
            "flex": 1,
          }
        }
      >
        <Text
          numberOfLines={1}
          style={
            Object {
              "color": "#000",
              "fontFamily": "System",
              "fontSize": 17,
              "fontWeight": "700",
              "marginLeft": undefined,
              "paddingLeft": 4,
              "paddingTop": 1,
              "textAlign": "center",
            }
          }
        >
          Header
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "flex-end",
            "alignSelf": "center",
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "flex-end",
          }
        }
      />
    </View>
  </View>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Array [
        Object {
          "padding": undefined,
        },
        undefined,
      ]
    }
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutomaticScroll={true}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardShouldPersistTaps="handled"
    keyboardSpace={0}
    onScroll={[Function]}
    resetKeyboardSpace={[Function]}
    resetScrollToCoords={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
    viewIsInsideTabBar={false}
  >
    <View>
      <Text
        style={
          Object {
            "color": "#000",
            "fontFamily": "System",
            "fontSize": 16,
          }
        }
        uppercase={false}
      >
        This is Content Section
      </Text>
    </View>
  </RCTScrollView>
  <View
    style={
      Object {
        "backgroundColor": "#F8F8F8",
        "borderColor": "#cbcbcb",
        "borderTopWidth": 0.5,
        "elevation": 3,
        "flexDirection": "row",
        "height": 55,
        "justifyContent": "center",
        "left": 0,
        "paddingBottom": 0,
        "right": 0,
      }
    }
  >
    <View
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": undefined,
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "alignSelf": "stretch",
            "backgroundColor": "transparent",
            "borderBottomWidth": null,
            "borderColor": null,
            "borderLeftWidth": null,
            "borderRadius": 0,
            "borderRightWidth": null,
            "borderTopWidth": null,
            "borderWidth": undefined,
            "elevation": 0,
            "flex": 1,
            "flexDirection": null,
            "height": 55,
            "justifyContent": "center",
            "opacity": 1,
            "paddingBottom": 6,
            "paddingTop": 6,
            "shadowColor": null,
            "shadowOffset": null,
            "shadowOpacity": null,
            "shadowRadius": null,
          }
        }
      >
        <Text
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#6b6b6b",
              "fontFamily": "System",
              "fontSize": 14,
              "lineHeight": 16,
              "marginLeft": 0,
              "marginRight": 0,
              "paddingLeft": 16,
              "paddingRight": 16,
            }
          }
          uppercase={false}
        >
          Footer
        </Text>
      </View>
    </View>
  </View>
</View>
`;
